src/api/subscriptions/setSubscriptionProperty.js

```diff
@@ -2,11 +2,18 @@
 import type { ApiResponse, Auth } from '../transportTypes';
 import { apiPost } from '../apiFetch';
 
-export default async (auth: Auth, streamId: number, value: boolean): Promise<ApiResponse> =>
+export type SubscriptionProperty = 'in_home_view' | 'pin_to_top' | 'push_notifications';
+
+export default async (
```

Comment:
Let's give this a quick bit of jsdoc that points to the API documentation:
https://chat.zulip.org/api/update-subscription-settings

That has quite a high ratio of helpfulness to effort, so it's a good minimal form of documentation for these API bindings. (And is really all they need when they don't have significant wrinkles of their own to document -- no sense duplicating what's in the server API docs.)